test/defaults/without-config-and-entry/default-without-config.test.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+const { stat } = require('fs');
+const { resolve } = require('path');
+const { run, extractSummary } = require('../../utils/test-utils');
+
+describe('output flag defaults without config', () => {
+    it('should throw if the entry file is not present', done => {
+        const { stdout, stderr } = run(__dirname, ['--defaults'], false);
+        const summary = extractSummary(stdout);
+        const outputDir = 'without-config-and-entry/dist';
+        // eslint-disable-next-line quotes
```

Comment:
Remove this line. Use the template literal (`)

Reply:
Cool, will do 👍

Reply:
Actually we're enforcing single quotes in eslint config.